airflow/models/taskinstance.py

```diff
@@ -3835,10 +3833,12 @@ def __init__(self, content, user_id=None):
         self.user_id = user_id
 
     def __repr__(self):
-        prefix = f"<{self.__class__.__name__}: {self.dag_id}.{self.task_id} {self.run_id}"
-        if self.map_index != -1:
-            prefix += f" map_index={self.map_index}"
-        return prefix + ">"
+        if self.task_instance:
```

Comment:
Can this be none? Don't think so?

We should at least have the ti_id as well.

Reply:
Yeah, it can't be none since we used TI.id as the primary key which means it'd always have a value. I will update it